internal/backend/crypto/gpg/cli/gpg.go

```diff
@@ -33,6 +33,7 @@ type GPG struct {
 	pubKeys   gpg.KeyList
 	privKeys  gpg.KeyList
 	listCache *lru.TwoQueueCache
+	throwKids bool
```

Comment:
Poor kids 😆

Reply:
No kids were harmed in making this PR ;)